aclocal.m4

```diff
@@ -42,21 +42,28 @@ AC_DEFUN([OCAML_CC_VENDOR], [
   AC_PREPROC_IFELSE(
     [AC_LANG_SOURCE([
 #if defined(_MSC_VER)
-msvc _MSC_VER
++msvc _MSC_VER
 #elif defined(__INTEL_COMPILER)
-icc __INTEL_COMPILER
++icc __INTEL_COMPILER
 #elif defined(__clang_major__) && defined(__clang_minor__)
-clang __clang_major__ __clang_minor__
++clang __clang_major__ __clang_minor__
+#elif defined(__MINGW32__)
+#include <windows.h>
```

Comment:
It's necessary to include `<windows.h>` to get the `__MINGW64_VERSION_MAJOR` macro, which is necessary to detect whether the compiler is mingw-w64 or a very old mingw, which is not supported.

Obviously, including a header adds much more noise to the output, which is why the lines are now prefixed with `+` and processed with sed below.